packages/cli/src/services/orchestration/webhook/handle-command-message-webhook.ts

```diff
@@ -1,15 +1,17 @@
+import { Container } from 'typedi';
+import { InstanceSettings } from 'n8n-core';
+
 import { ExternalSecretsManager } from '@/external-secrets/external-secrets-manager.ee';
+import { Logger } from '@/logger';
 import { License } from '@/license';
 import { MessageEventBus } from '@/eventbus/message-event-bus/message-event-bus';
-import Container from 'typedi';
-import { Logger } from 'winston';
```

Comment:
not sure how this was even working considering that `winston.Logger` doesn't exist on the DI container.